retrace-server.spec.in

```diff
@@ -99,7 +100,42 @@ rm -f %{buildroot}%{_infodir}/dir
 #retrace uid/gid reserved in setup, rhbz #706012
 %define retrace_gid_uid 174
 getent group retrace > /dev/null || groupadd -f -g %{retrace_gid_uid} --system retrace
-getent passwd retrace > /dev/null || useradd --system -g retrace -u %{retrace_gid_uid} -d %{_datadir}/%{name} -s /sbin/nologin retrace
+getent passwd retrace > /dev/null || useradd --system -g retrace -u %{retrace_gid_uid} -b %{_sharedstatedir} -s /sbin/nologin retrace
+
+SUB_UID_MIN=$(grep -s '^SUB_UID_MIN' /etc/login.defs | grep -o '[0-9]\+' || echo "100000")
+SUB_UID_COUNT=$(grep -s '^SUB_UID_COUNT' /etc/login.defs | grep -o '[0-9]\+' || echo "65536")
+
+SUB_GID_MIN=$(grep -s '^SUB_GID_MIN' /etc/login.defs | grep -o '[0-9]\+' || echo "100000")
+SUB_GID_COUNT=$(grep -s '^SUB_GID_COUNT' /etc/login.defs | grep -o '[0-9]\+' || echo "65536")
+
+if [ -f /etc/subuid ]; then
+    if ! grep -q '^retrace:' /etc/subuid; then
+        LAST_SUBUID=$(cut -f 2-3 -d ':' /etc/subuid \
```

Comment:
Seems like this could be a tad problematic if the file is empty.

Reply:
Ah, yes, I assumed "empty" was a subset of "malformed" from your earlier comment. Silly of me not to think of `[ -s`. Fixed.

Reply:
If there are no regular users, it is very likely that the file will be empty, unless the server hosts more services that could have potentially added entries there.